src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -25,14 +23,10 @@ ifeval::["{project-context}" == "che"]
 Latest version: link:https://+github.com/eclipse/che/blob/master/deploy/openshift/templates/monitoring/prometheus-config.yaml[example `prometheus-config.yaml` on GitHub].
 endif::[]
 +
-There are two blocks of configuration in the `prometheus-config.yml` block of the example configuration file: `global` and `scrape_configs`.
-+
-The `global` block controls the Prometheus server's global configuration. There are two options present:
-
-* `scrape_interval` controls how often Prometheus scrapes targets. You can override this for individual targets. In this case, the global setting is to scrape every 5 seconds.
-* `evaluation_interval` controls how often Prometheus evaluates rules. Prometheus uses rules to create new time series and to generate alerts. This option is not used in the example here.
-+
-The last block, `scrape_configs`, controls what resources Prometheus monitors. In the default configuration, there is a single job called `{prod-host}`, which scrapes the time series data exposed by the {prod-short} server. The job contains a single, statically configured target, the `{prod-host}` host on port `8087`.
+<1> Rate, at which a target is scraped.
```

Comment:
Does this render correctly? I don't think the added line (Latest version:) can be between the code block and these call out descriptions.

Reply:
@rkratky renders like this when I build it locally: https://drive.google.com/open?id=10ZfiBgcDkzBNwMikX3pMEoZ2G6ojHTau

Reply:
Yep, same for me, Boss :) I think its ok, but let me know in the case that some tunning will have to be done! :)